src/main/java/org/broadinstitute/hellbender/tools/spark/sv/CallVariantsFromAlignedContigsSpark.java

```diff
@@ -0,0 +1,444 @@
+                .attribute(GATKSVVCFHeaderLines.TOTAL_MAPPINGS, numAssembledBreakpoints)
+                .attribute(GATKSVVCFHeaderLines.HQ_MAPPINGS, highMqMappings)
+                .attribute(GATKSVVCFHeaderLines.MAPPING_QUALITIES, mqs.stream().map(String::valueOf).collect(Collectors.joining(",")))
+                .attribute(GATKSVVCFHeaderLines.ALIGN_LENGTHS, alignLengths.stream().map(String::valueOf).collect(Collectors.joining(",")))
+                .attribute(GATKSVVCFHeaderLines.MAX_ALIGN_LENGTH, maxAlignLength)
+                .attribute(GATKSVVCFHeaderLines.ASSEMBLY_IDS, breakpointIds.stream().collect(Collectors.joining(",")))
+                .attribute(GATKSVVCFHeaderLines.CONTIG_IDS, assembledContigIds.stream().map(s -> s.replace(" ", "_")).collect(Collectors.joining(",")));
+
+        if (breakpointAllele.insertedSequence.length() > 0) {
+            vcBuilder = vcBuilder.attribute(GATKSVVCFHeaderLines.INSERTED_SEQUENCE, breakpointAllele.insertedSequence);
+        }
+
+        if (breakpointAllele.insertionMappings.size() > 0) {
+            vcBuilder = vcBuilder.attribute(GATKSVVCFHeaderLines.INSERTED_SEQUENCE_MAPPINGS, breakpointAllele.insertionMappings.stream().collect(Collectors.joining(";")));
+        }
+
+        if (breakpointAllele.homology.length() > 0) {
+            vcBuilder = vcBuilder.attribute(GATKSVVCFHeaderLines.HOMOLOGY, breakpointAllele.homology);
+            vcBuilder = vcBuilder.attribute(GATKSVVCFHeaderLines.HOMOLOGY_LENGTH, breakpointAllele.homology.length());
+        }
+
+        if (breakpointAllele.getInversionType() == INV_5_TO_3) {
+            vcBuilder = vcBuilder.attribute(GATKSVVCFHeaderLines.INV_5_TO_3, "");
+        }
+
+        if (breakpointAllele.getInversionType() == INV_3_TO_5) {
+            vcBuilder = vcBuilder.attribute(GATKSVVCFHeaderLines.INV_3_TO_5, "");
+        }
+
+
+        return vcBuilder.make();
+    }
+
+
+    private static String getInversionId(final BreakpointAllele breakpointAllele) {
+        final String invType = breakpointAllele.getInversionType().name();
+        return invType + "_" + breakpointAllele.leftAlignedLeftBreakpoint.getContig() + "_" + breakpointAllele.leftAlignedLeftBreakpoint.getStart() + "_" + breakpointAllele.leftAlignedRightBreakpoint.getStart();
+    }
+
+
+    static Tuple2<BreakpointAllele, Tuple2<Tuple2<String,String>, BreakpointAlignment>> keyByBreakpointAllele(final Tuple2<Tuple2<String, String>, BreakpointAlignment> breakpointIdAndAssembledBreakpoint) {
+        final BreakpointAllele breakpointAllele = breakpointIdAndAssembledBreakpoint._2.getBreakpointAllele();
+        return new Tuple2<>(breakpointAllele, new Tuple2<>(breakpointIdAndAssembledBreakpoint._1, breakpointIdAndAssembledBreakpoint._2));
+    }
+
+    static Boolean inversionBreakpointAlleleFilter(final Tuple2<BreakpointAllele, Tuple2<Tuple2<String, String>, BreakpointAlignment>> breakpointAlleleTuple2Tuple2) {
```

Comment:
IMO, something like `breakpointAlleleRepresentsInversion` is clearer because this method is testing if the breakpoint allele really represents an inversion, not trying to further filter the inversion breakpoint alleles.

A related minor issue is that the method should preferably take a single argument of `BreakpointAllele`, because the associated `BreakpointAlignment` is not used. Of course if this method is intended to be extended in the future, and the alignments may be needed, this should not be changed.